docs/sql/data_types/timestamp.md

```diff
@@ -1,25 +1,59 @@
 ---
 layout: docu
 title: Timestamp Types
-blurb: A timestamp specifies a combination of a date (year, month, day) and a time (hour, minute, second, microsecond or nanosecond).
+blurb: Timestamps represent points in time.
 ---
 
-Timestamps represent points in absolute time, usually called *instants*.
-DuckDB represents instants as the number of microseconds (µs) (or nanoseconds, for `TIMESTAMP_NS`) since `1970-01-01 00:00:00+00`.
+Timestamps represent points in time and are internally stored as the `INT64` number of seconds / milliseconds / microseconds / nanoseconds since `1970-01-01 00:00:00+00`, depending on the chosen variant. Informally speaking, they contain both [`DATE`]({% link docs/sql/data_types/date.md %}) (year, month, day) and [`TIME`]({% link docs/sql/data_types/time.md %}) (hour, minute, second, microsecond or nanosecond) information. 
 
 ## Timestamp Types
 
 | Name | Aliases | Description |
 |:---|:---|:---|
-| `TIMESTAMP_NS` |                            | timestamp with nanosecond precision (ignores time zone)  |
-| `TIMESTAMP`    | `DATETIME`                 | timestamp with microsecond precision (ignores time zone) |
-| `TIMESTAMP_MS` |                            | timestamp with millisecond precision (ignores time zone) |
-| `TIMESTAMP_S`  |                            | timestamp with second precision (ignores time zone)      |
-| `TIMESTAMPTZ`  | `TIMESTAMP WITH TIME ZONE` | timestamp with microsecond precision (uses time zone)                               |
+| `TIMESTAMP_NS` |                                           | timestamp with nanosecond precision (local semantics)    |
```

Comment:
"ignores time zone" was correct!  I don't know what "local semantics" means, but if you want to be precise, UTC would be more accurate. To be local, you would need a time zone.